R/setAccess.R

```diff
@@ -60,8 +61,7 @@ access = match.arg(access, c('public','internal','cooperator','USGS'))
   pkg.env$wqpMetrics = "https://www.waterqualitydata.us/ActivityMetric/search"
   
   pkg.env$NGWMN = "https://cida.usgs.gov/ngwmn_cache/sos"
-  
-  options(Access.dataRetrieval = access)
+
```

Comment:
To avoid confusion that the user thinks they can set an option to internal/public etc. The user can still get the info by looking at the attr(x, "url"), or they can run `setAccess`. I'm unaware of anyone using options(Access.dataRetrieval).